asserts/toolbelt/main.go

```diff
@@ -38,9 +38,20 @@ var parser = flags.NewParser(nil, flags.Default)
 var db *asserts.Database
 
 func main() {
-	var err error
+	topDir := "snappy-asserts-toolbelt-db"
+	bs, err := asserts.OpenFilesystemBackstore(topDir)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	keypairMgr, err := asserts.OpenFilesystemKeypairManager(topDir)
```

Comment:
@niemeyer not sure they are used often enough to warrant the compressiong, I looked around we are not super consistent in using FS vs Filesystem